yt/chyt/controller/internal/jupyt/api.go

```diff
@@ -0,0 +1,54 @@
+package jupyt
+
+import (
+	"context"
+	"fmt"
+	"go.ytsaurus.tech/yt/chyt/controller/internal/api"
+	"go.ytsaurus.tech/yt/chyt/controller/internal/strawberry"
+	"go.ytsaurus.tech/yt/go/yt"
+	"go.ytsaurus.tech/yt/go/yterrors"
+	"net"
+)
+
+type GetEndpointResult struct {
+	Address     string         `yson:"address"`
+	OperationId yt.OperationID `yson:"operation_id"`
+	JobId       yt.JobID       `yson:"job_id"`
+}
+
+func GetEndpoint(a *api.API, ctx context.Context, alias string) (result GetEndpointResult, err error) {
+	var briefInfo strawberry.OpletBriefInfo
```

Comment:
let's use short variable declaration with := instead of var briefInfo, var jobs, var address; other things lgtm